varcode/variant_collection.py

```diff
@@ -161,10 +124,10 @@ def effects_to_string(self, *args, **kwargs):
         Arguments are passed on to variant_effects(*args, **kwargs).
         """
         lines = []
-        for variant_effect in self.variant_effects(*args, **kwargs):
+        for annotation in self.annotations(*args, **kwargs):
```

Comment:
header comment needs to be updated for new "annotation" language. Consider renaming method to something like `annotations_summary`?